README.md

```diff
@@ -93,9 +94,11 @@ Legend:
 
 At present the tool will show an error if it cannot connect to the repository for an image for any reason.
 This could include:
```

Comment:
Please update the example to include some errors being detected, and change this section to say something like:
> if there is an error checking an image the tool will display the 🚫 symbol and give a short description of the error at the end of the line.